games-puzzle/gottet/gottet-1.2.3.recipe

```diff
@@ -21,37 +21,49 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
+	qt6_tools$secondaryArchSuffix
```

Comment:
qt6_tools aren't usually required at run time, just build time. Specifying the qt6_tools_devel package in BUILD_REQUIRES like you've done below should be enough since it will pull in the main qt6_tools package as well.
```suggestion
```